datafusion/core/tests/sql/order.rs

```diff
@@ -1,234 +0,0 @@
-        "+-----+--------+",
-        "| 2   | two    |",
-        "| 1   | one    |",
-        "|     | three  |",
-        "+-----+--------+",
-    ];
-    assert_batches_eq!(expected, &actual);
-    Ok(())
-}
-
-#[tokio::test]
-async fn test_specific_nulls_first_asc() -> Result<()> {
-    let ctx = SessionContext::new();
-    let sql = "SELECT * FROM (VALUES (1, 'one'), (2, 'two'), (null, 'three')) AS t (num,letter) ORDER BY num ASC NULLS FIRST";
-    let actual = execute_to_batches(&ctx, sql).await;
-    let expected = vec![
-        "+-----+--------+",
-        "| num | letter |",
-        "+-----+--------+",
-        "|     | three  |",
-        "| 1   | one    |",
-        "| 2   | two    |",
-        "+-----+--------+",
-    ];
-    assert_batches_eq!(expected, &actual);
-    Ok(())
-}
-
-#[tokio::test]
-async fn sort() -> Result<()> {
-    let results =
-        partitioned_csv::execute("SELECT c1, c2 FROM test ORDER BY c1 DESC, c2 ASC", 4)
-            .await?;
-    assert_eq!(results.len(), 1);
-
-    let expected: Vec<&str> = vec![
-        "+----+----+",
-        "| c1 | c2 |",
-        "+----+----+",
-        "| 3  | 1  |",
-        "| 3  | 2  |",
-        "| 3  | 3  |",
-        "| 3  | 4  |",
-        "| 3  | 5  |",
-        "| 3  | 6  |",
-        "| 3  | 7  |",
-        "| 3  | 8  |",
-        "| 3  | 9  |",
-        "| 3  | 10 |",
-        "| 2  | 1  |",
-        "| 2  | 2  |",
-        "| 2  | 3  |",
-        "| 2  | 4  |",
-        "| 2  | 5  |",
-        "| 2  | 6  |",
-        "| 2  | 7  |",
-        "| 2  | 8  |",
-        "| 2  | 9  |",
-        "| 2  | 10 |",
-        "| 1  | 1  |",
-        "| 1  | 2  |",
-        "| 1  | 3  |",
-        "| 1  | 4  |",
-        "| 1  | 5  |",
-        "| 1  | 6  |",
-        "| 1  | 7  |",
-        "| 1  | 8  |",
-        "| 1  | 9  |",
-        "| 1  | 10 |",
-        "| 0  | 1  |",
-        "| 0  | 2  |",
-        "| 0  | 3  |",
-        "| 0  | 4  |",
-        "| 0  | 5  |",
-        "| 0  | 6  |",
-        "| 0  | 7  |",
-        "| 0  | 8  |",
-        "| 0  | 9  |",
-        "| 0  | 10 |",
-        "+----+----+",
-    ];
-
-    // Note it is important to NOT use assert_batches_sorted_eq
-    // here as we are testing the sortedness of the output
-    assert_batches_eq!(expected, &results);
-
-    Ok(())
-}
-
-#[tokio::test]
-async fn sort_empty() -> Result<()> {
-    // The predicate on this query purposely generates no results
-    let results = partitioned_csv::execute(
-        "SELECT c1, c2 FROM test WHERE c1 > 100000 ORDER BY c1 DESC, c2 ASC",
-        4,
-    )
-    .await
-    .unwrap();
-    assert_eq!(results.len(), 0);
-    Ok(())
-}
-
-#[tokio::test]
-async fn sort_with_lots_of_repetition_values() -> Result<()> {
-    let ctx = SessionContext::new();
-    let filename = "tests/parquet/data/repeat_much.snappy.parquet";
-
-    ctx.register_parquet("rep", filename, ParquetReadOptions::default())
-        .await?;
-    let sql = "select a from rep order by a";
```

Comment:
Should this test be moved to .slt as well?

Reply:
Yes. I am not sure how I missed this . Thank you for the catch. It appears to have a pretty massive output that is not easy to replicate in slt, so I will leave it in rust for now.

 05fb04960